### PR TITLE
feedback: Fix table columns

### PR DESCRIPTION
**Pull request by c298lee**
Fixes table columns, using \` causes the escape character to show up so using code instead.

Before:
Screenshot 2024-01-18 at 4 27 15 PM
After:
Screenshot 2024-01-18 at 4 19 28 PM

### CODE REVIEW

File: src/platforms/javascript/common/user-feedback/configuration/index.mdx [hunk 1]
```diff
@@ -16,11 +16,11 @@ The User Feedback Widget offers many customization options, and if the available
...
+| `autoInject`   | `boolean`                                  | `true`     | Injects the Feedback widget into the application when the integration is added. Set `autoInject: false` if you want to call `feedback.attachTo()` or `feedback.openDialog()` directly, or only want to show the widget on certain views. |
+| `showBranding` | `boolean`                                  | `true`     | Displays the Sentry logo inside of the form                                                                                                                                                                                              |
+| `colorScheme`  | <code>"system" \| "light" \| "dark"</code> | `"system"` | The color theme to use. `"system"` will follow your OS colorscheme.                                                                                                                                                                      |
```
Comment (michellewzhang) [inline src/platforms/javascript/common/user-feedback/configuration/index.mdx:23]:
```suggestion
| `colorScheme`  | <code>"system" \| "light" \| "dark"</code> | `"system"` | The color theme to use. `"system"` will follow your OS color scheme.                                                                                                                                                                      |
```

File: src/platforms/node/common/user-feedback/configuration/index.mdx [hunk 2]
```diff
@@ -14,11 +14,11 @@ The User Feedback Widget offers many customization options, and if the available
...
+| `autoInject`   | `boolean`                                  | `true`  | Injects the Feedback widget into the application when the integration is added. Set `autoInject: false` if you want to call `feedback.attachTo()` or `feedback.openDialog()` directly, or only want to show the widget on certain views. |
+| `showBranding` | `boolean`                                  | `true`  | Displays the Sentry logo inside of the form                                                                                                                                                                                              |
+| `colorScheme`  | <code>"system" \| "light" \| "dark"</code> |         | `"system"`                                                                                                                                                                                                                               | The color theme to use. `"system"` will follow your OS colorscheme. |
```
Comment (michellewzhang) [inline src/platforms/node/common/user-feedback/configuration/index.mdx:21]:
```suggestion
| `colorScheme`  | <code>"system" \| "light" \| "dark"</code> |         | `"system"`                                                                                                                                                                                                                               | The color theme to use. `"system"` will follow your OS color scheme. |
```

**Review by michellewzhang — APPROVED** [review]

File: src/platforms/node/common/user-feedback/configuration/index.mdx [hunk 3]
```diff
@@ -14,11 +14,11 @@ The User Feedback Widget offers many customization options, and if the available
...
-| `colorScheme`  | `"system" | "light" | "dark"`                                                                                                                                                                                                                                  | `"system"` | The color theme to use. `"system"` will follow your OS colorscheme. |
+| key            | type                                       | default | description                                                                                                                                                                                                                              |
+| -------------- | ------------------------------------------ | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------- |
```
Comment (vivianyentran) [inline src/platforms/node/common/user-feedback/configuration/index.mdx:18]:
```suggestion
| -------------- | ------------------------------------------ | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | 
```

File: src/platforms/node/common/user-feedback/configuration/index.mdx [hunk 4]
```diff
@@ -14,11 +14,11 @@ The User Feedback Widget offers many customization options, and if the available
...
+| `autoInject`   | `boolean`                                  | `true`  | Injects the Feedback widget into the application when the integration is added. Set `autoInject: false` if you want to call `feedback.attachTo()` or `feedback.openDialog()` directly, or only want to show the widget on certain views. |
+| `showBranding` | `boolean`                                  | `true`  | Displays the Sentry logo inside of the form                                                                                                                                                                                              |
+| `colorScheme`  | <code>"system" \| "light" \| "dark"</code> |         | `"system"`                                                                                                                                                                                                                               | The color theme to use. `"system"` will follow your OS color scheme. |
```
Comment (vivianyentran) [inline src/platforms/node/common/user-feedback/configuration/index.mdx:21]:
```suggestion
| `colorScheme`  | <code>"system" \| "light" \| "dark"</code> | `"system"`                                                                                                                                                                                                                               | The color theme to use. `"system"` will follow your OS color scheme. |
```

File: src/platforms/node/common/user-feedback/configuration/index.mdx (hunk 3 above)
Comment (vivianyentran) [reply src/platforms/node/common/user-feedback/configuration/index.mdx:18]: I'm seeing 5 columns here in the preview, so hope this fixes it!

**Review by vivianyentran — APPROVED** [review]
Looks good now! Thanks for fixing :)

**Pull request merged**